test/test_proxy_tensor.py

```diff
@@ -1337,11 +1330,15 @@ def f(a, b, c, d, e):
 
 symbolic_tensor_failures.update(symbolic_tensor_segfaults)
 
+outplace_symbolic_tensor_failures = {
+    xfail('masked_fill', ''),  # expected predicate to be bool, got torch.float32
+    xfail('masked_scatter', ''),  # aten.masked_scatter.default - couldn't find symbolic meta function/decomposition
+    xfail('nn.functional.rrelu', ''),  # aten.empty_like.default - couldn't find symbolic meta function/decomposition
+}
+
 inplace_symbolic_tensor_failures = {
-    xfail('abs', ''),  # aten.abs_.default - couldn't find symbolic meta function/decomposition
```

Comment:
oh because they now use python decomps instead of C++ decomps?

Reply:
Specifically, I believe these started passing because previously they went through c++ composite which was not symint aware, and now they fall through to meta + fake tensor which does work. But... I didn't verify this lol